perf/bench/0c34/prep

```diff
@@ -0,0 +1,38 @@
+#! /bin/sh
+#
+# Fetch rules and targets prior for the "r2c" benchmark.
+#
+# rule_dir: input/golang.yaml
+# target_dir: input/govwa
+#
+# Uses sh because bash is not installed within the semgrep docker container.
+#
+set -eu
+
+mkdir -p input
+
+cp golang.yml input/
```

Comment:
please, explain how you generated this golang.yml file. Did you curl
a certain r/golang? then please put the command in comment, so if we update the golang rules we know how to update golang.yml